sema/bench/__main__.py

```
@@ -0,0 +1,108 @@
+    args = get_arg_parser().parse_args(args_list)
+
+    try:
+        sembench = make_service(args)
+        sembench.run()
+    except Exception as e:
+        log.exception("sema.bench processing failed", exc_info=e)
+        return False
+    return True
+
+
+def main():
```

Ruff

Missing return type annotation for public function `main`

Add return type annotation: `None`

(ANN201)